docs/config.md

```diff
@@ -13,7 +13,7 @@ Settings within section names in the config just require you to prefix the secti
 Examples:
 
 ```
-MT_LOG_LEVEL: 1                           # MT_<setting_name>
+MT_LOG_LEVEL: info                        # MT_<setting_name>
```

Comment:
in same places in this PR you're replacing `2` with `info`, and here you replace `1` with `info`. are they the same or should that be something else?

Reply:
'1' would be the new equivalent of 'debug', however that is a discrepancy between the documentation and the actual config / example config, so I thought it best to change the docs to match up with the config files themselves. Otherwise it might lead to confusion if someone looks at the docs and just assumes that the configs match them.